src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -486,27 +487,46 @@ def add_digital_object(self, parent_archival_object, dashboard_uuid, title="", i
             "notes": [{
                 "jsonmodel_type": "note_digital_object",
                 "type": "originalsloc",
-                "content": [dashboard_uuid]
+                "content": [dashboard_uuid],
+                "publish": True,
             }],
             "restrictions": restricted,
             "subjects": parent_record['subjects'],
             "linked_agents": parent_record['linked_agents'],
         }
 
+        i = 0
```

Comment:
Is this count used?

If you want to get a count when iterating over an enumerable, you can use the `enumerate` function:

```python
for i, pnote in enumerate(parent_record["notes"]):
    ...
```